src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Parser/KnownTypeSymbols.cs

```diff
@@ -120,7 +125,11 @@ public KnownTypeSymbols(CSharpCompilation compilation)
             Enum = compilation.GetSpecialType(SpecialType.System_Enum);
             ArgumentNullException = compilation.GetBestTypeByMetadataName(typeof(ArgumentNullException));
 
-            String = compilation.GetBestTypeByMetadataName(typeof(string));
```

Comment:
Was this removal intentional?  If so shouldn't the property be removed as well?

Reply:
Also, if we do end up keeping this it should be initialized using `SpecialType.System_String` -- resolving by metadata name can be expensive.

Reply:
If you scroll up a little you will find it is already defined 
```C#
            String = compilation.GetSpecialType(SpecialType.System_String);
```

I have no idea why we defined it twice.